tests/minifier.js

```diff
@@ -3075,6 +3074,14 @@ QUnit.test('sort style classes', function(assert) {
   assert.equal(minify(input, { sortClassName: false }), input);
   output = '<a class="add createSorter keys sort"></a>';
   assert.equal(minify(input, { sortClassName: true }), output);
+
+  input = '<span class="sprite sprite-{{sprite}}"></span>\n<span class="{{sprite}}"></span>';
+  output = '<span class="sprite sprite-{{sprite}}"></span> <span class="{{sprite}}"></span>';
+  assert.equal(minify(input, {
+    collapseWhitespace: true,
+    ignoreCustomFragments: [/{{.*}}/],
```

Comment:
I'm well aware of that, just didn't worry too much as this is a single test case of specific input.

As you can see in https://github.com/kangax/html-minifier#options-quick-reference, the default values for `ignoreCustomFragments` is not only non-greedy, but can also handle multi-line values in between the template markers.

Nonetheless, I'll add the `?` to make it easier for your eyes. :wink: